solidity/migrations/2_sausage_migration.js

```diff
@@ -1,20 +1,143 @@
 const SausageTokens = artifacts.require('./SausageTokens.sol')
 const Auction = artifacts.require('./Auction.sol')
 
-module.exports = async function(deployer) {
-  await deployer.deploy(SausageTokens, 'SausageTokens', 'Bratwurst')
-  const sausage = await SausageTokens.deployed()
-  await deployer.deploy(
-    Auction,
-    '0xe207259495d9de39e92e8b280a85b4d845906ce6',
-    1234,
-    'Bratwurst'
-  )
-  const auction = await Auction.deployed()
-}
+const Web3 = require('web3')
+
+module.exports = async function(deployer, network) {
+  let web3
+  let sausageID = 1234
+  let sausageSymbol = 'Bratwurst'
+  let sausageDescription = 'Bratwurst - Rare'
+  let sausageInstance
+  let managerAccount
+  let accounts
+
+  if (network === 'development') {
+    web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'))
+    managerAccount = web3.accounts[0]
+  } else if (network === 'kovan') {
+    // do something similar
+  } else if (network === 'parity') {
+    web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'))
+    await web3.eth.getAccounts().then(acc => {
```

Comment:
i think we can get manager account easier. see polymath https://github.com/PolymathNetwork/polymath-core/blob/master/migrations/2_deploy_contracts.js#L27-L34 line 27 and line 34 show you can pass accounts directly into the function/ we can do this for all network (parity, development, and the others once they are created)